pkg/shell/manifest.json

```diff
@@ -34,6 +34,7 @@
     ],
     "config": {
         "host_switcher": {
+            "platform:f39": true,
             "platform:el9": true,
             "bookworm": true,
             "jammy": true
```

Comment:
need to add noble as well

Reply:
done

Reply:
ah, sorry, I thought jammy was 2204 because testlib.py already sets multihost_enabled for ubuntu-2204... did I mention I am confused by Ubuntu versions?

Reply:
Ah, **24**04, of course!

Reply:
Which we call.... ubuntu-stable?